reactioncommerce/reaction-component-library
# feat: added text overrides on CartItemDetail

Part of: #409 
Impact: **minor**  
Type: **feature**

## Component
CartItemDetail now has overrides when i18next will be implemented (see #409)

## Breaking changes
No breaking chances, all added fields are optional

## Testing
1. Add a new prop `quantityText` to `<CartItemDetail />` 
2. Add the value `"test"` to `quantityText`
3. Label should display "test" when test is given as a value to the prop. When the prop `quantityText` is not added, it should default to the values given in default props
